oreoluwade/dms-client-react
# [Feature] Routing, Header, Landing page

### What does this PR do?
- Configure routing, create Header and Landing Page components

### Description of tasks accomplished
- Setup routing
- Creating Landing Page and Header components
- FIx hot reloading mishap

### How to test?
- Start the App
- See all features properly configured
